marketplace-solver/src/api.rs

```diff
@@ -101,8 +102,16 @@ where
     api.post("submit_bid", |_req, _state| {
         async move { Ok("Bid Submitted") }.boxed()
     })?
-    .get("auction_results", |_req, _state| {
-        async move { Ok("Auction Results Gotten") }.boxed()
+    .get("auction_results", |req, _state| {
+        async move {
```

Comment:
I forgot to call the `calculate_auction_results_permissionless` from the state here. I can push the change if you are okay w that

Reply:
or I think I'll create a separate PR for it and merge it into main

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1843